res/raw/webxdc_wrapper.html

```diff
@@ -129,8 +129,6 @@
           } else {
             loadingDiv.innerHTML = "";
             iframe.src = "index.html";
-            iframe.contentWindow.webxdc_internal = window.webxdc_internal;
```

Comment:
`webxdc_internal` seems not to be used at all, grep returns zero results for me.

Reply:
that comes from desktop, can be removed on android